report/2.Analysis/Requirements.tex

```diff
@@ -0,0 +1,39 @@
+\section{Requirements}
+In order to be able to conclude upon the success of the project, a fixed set of requirements must be formulated.
+These requirements will serve as a guideline in the conclusion of the report and as markers throughout the project. 
+To formally specify the requirements, the MoSCoW analysis model will be utilized. 
+The MoSCoW model specifies which requirements \textbf{M}ust, \textbf{S}hould, \textbf{C}ould, and \textbf{W}on't be fulfilled to deem the project successful.
+
+\textbf{Must have}
+\begin{itemize}
+\item ability to recognize a moving target in front of the device using an attached camera
+\item ability to hit a target moving with a constant speed along a horizontal axis
+\item move fast enough to keep the laser on the moving target in real time
```

Comment:
her skal specifieres en rimelig grænse for hvor langt væk. vi kan ikke følge med hvis bolden er 5 centimeter væk. måske skrive i den øverste del af disse requirements går ud fra at målet bevæger sig med X hastighed cirka 2 meter fra device ... eller lignende

Reply:
Jeg tænker at i must have er den fin med at være lidt vague, det gør at hvis vi kun bliver i stand til at tracke et object, eg. en legobil, med et super forudsigeligt bevægelsesmønster er den opfyldt, da vi har en grundfunktionalitet.

Den bliver så udvidet i should have med moving target - den har jeg derimod givet en mere bestemt grænse.